### PR TITLE
Added Microsoft.AlertsManagement/resourceHealth

### PR DESCRIPTION
**Pull request by giladsu**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2020-08-04-preview/AlertsManagement.json [hunk 1]
```diff
@@ -0,0 +1,649 @@
...
+        "enabled",
+        "description",
+        "criteria"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "the description of the health alert that will be included in the alert email."
+        },
+        "enabled": {
```
Comment (pilor) [inline specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2020-08-04-preview/AlertsManagement.json:384]: strongly encourage using "alertState": "Enabled"/"Disabled" (or similar) with a string enum instead to allow for future states

**Review by pilor — APPROVED** [review]
1 suggestion

**Comment by qianwens** [comment]
@giladsu is the PR ready to merge, please let us know if it is an ignite feature.

**Comment by giladsu** [comment]
> @giladsu is the PR ready to merge, please let us know if it is an ignite feature.

@qianwens - this PR is ready to merge for private preview. It is not for Ignite.
Thanks.

File: specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2020-08-04-preview/AlertsManagement.json (hunk 1 above)
Comment (giladsu) [reply specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2020-08-04-preview/AlertsManagement.json:384]: Thanks @pilor - we would like this to be aligned with current metric alerts API in order not to cause confusion with customers...

**Pull request merged**